src/views/Expenses.js

```diff
@@ -1,66 +1,98 @@
 import AddExpensesForm from '../components/AddExpensesForm';
 import ExpensesList from '../components/ExpensesList';
+import ExpenseAddedAlert from '../components/ExpenseAddedAlert';
+import ExpenseDeletedAlert from '../components/ExpenseDeletedAlert';
+import { useState } from 'react';
 
 const Expenses = ({
-  selectedMonth,
   expenses,
   setExpenses,
   filteredExpenses,
   setSelectedMonth,
-  currentMonth
+  currentMonth,
 }) => {
+  // `expenseAdded` and `expenseDeleted` can be undefined, true or false
+  // alerts are hidden when these are undefined
+  const [expenseAdded, setExpenseAdded] = useState();
```

Comment:
question: does bootstrap offers anything to handle alert events? (I do not see any problem in your approach, just wondering whether bootstrap can do it for us, it might require using the JS bundle they offer, not sure thou)

Reply:
Actually yeah there's an alert plugin, but it looks like it relies on direct DOM manipulation. I would assume that we want to handle triggering and rendering in a "react" way. There might be a more elegant solution in something like React-Bootstrap. If you have any ideas, let me know!